CONTRIBUTING.md

```diff
@@ -0,0 +1,114 @@
+on the quality and standards of submitted contributions.
+
+### Open an Issue on GitHub
+
+The Gamepad Navigator uses [GitHub issues](https://github.com/fluid-lab/gamepad-navigator/issues) to keep track of the
+open tickets. The branch containing your contribution should be named after the GitHub issue number you're working
+on. For example, branch "GH-28" for issue #28.
+
+GitHub issues should be meaningful and describe the task, bug, or feature in a way that can be understood by the
+community. Opaque or general descriptions should be avoided. If you have a large task that will involve several
+substantial commits, consider breaking it up into subtasks and mention them in the ticket's description. That way, you
+can create multiple pull requests that are easier to review. For example, break issue #28 into two subtasks and make
+a branch for each subtask, such as "GH-28-1" and "GH-28-2".
+
+<!-- TODO: Add an issue and pull request template on GitHub -->
+Go through the following steps to open a new issue/ticket:
+
+1. Search through the existing tickets to see if a similar ticket is already filed. If you find a ticket that seems to
+   match yours, add a comment to the same ticket describing any additional information you have (different context or
+   environment, speculated cause, possible solution, or any other feedback). Otherwise, create a new ticket.
+2. Provide as much relevant detail in the ticket description. If the ticket is a bug report, it should include the
+   following:
+   - Details of the actual behavior.
+   - Instructions for reproducing that bug.
+   - Difference between the actual and expected behavior.
+   - Additional details about the general context.
+3. Besides, more technical information can be added that might help whosoever working on that ticket (see below).
+   - Console logs or error messages.
+   - Suspected technical cause of the problem.
+   - Suggestions for resolving that bug.
+
+### Add Unit Tests
+
+Production-level code needs to be accompanied by a reasonable suite of unit tests. This helps others confirm that the
+code is working as intended, and allows the community to adopt more agile refactoring techniques while being more
+confident in our ability to avoid regressions. All unit tests should be written using jqUnit (check the
+[jqUnit documentation](https://docs.fluidproject.org/infusion/development/jqUnit.html) for more information).
+
+### Lint Your Code
+
+JavaScript is a highly dynamic and loose language, and many common errors are not picked up until run time. To avoid
+errors and common pitfalls in the language, and to maintain consistency in terms of syntax, lint your code regularly
+using the provided Grunt lint task (see below). Ensure that you run lint checks on your code before making a commit.
+
+``` snippet
+# Runs linting tasks
+grunt lint
+```
+
+### Create Meaningful Commit Logs
+
+All commits logs should include the following information:
```

Comment:
"commits" shouldn't be plural, but I'd suggest "commit log messages" as that's what you really mean.